src/spans/awsSpan.ts

```diff
@@ -328,14 +318,16 @@ export const getHttpSpan = (
 
   const prioritizedSpanId = getHttpSpanId(randomRequestId, spanId);
   if (requestData) {
-    requestData.body && (requestData.body = shallowMask('requestBody', requestData.body));
+    requestData.body &&
```

Comment:
i would appreciate using full if conditions for readability

Reply:
There is no logical difference with the code that was already merged, but sure, I'll send a patch